include/librpmi_env.h

```diff
@@ -80,6 +80,27 @@ static inline void *rpmi_env_memcpy(void *dest, const void *src, rpmi_size_t n)
 	return dest;
 }
 
+/**
+ * @brief Write a hw memory range with buffer
+ *
+ * @param[in] dest 	destination hw address
+ * @param[in] buf 	buffer pinter
+ * @param[in] n 	number of bytes to write
+ * @return void *	Pointer to destination
+ */
+void *rpmi_env_mem_write(char *dest, const void *buf, rpmi_size_t n);
```

Comment:
`char` can be `void` and leaving the granularity of write/read to the host firmware.